templates/overrides/infrastructure-vsphere/v0.6.4/cluster-template-external-loadbalancer.yaml

```diff
@@ -0,0 +1,178 @@
+---
+apiVersion: cluster.x-k8s.io/v1alpha3
```

Comment:
we are generating templates using this [golang tool](https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/tree/master/packaging/flavorgen) it basically generates the cluster templates programmatically

Reply:
ok, didn't see that. Will have a look at it when I get a gap.